components/CoverletterInformation.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <address>
+    <div 
+      class="company">
+      <span 
+        v-if="title"
```

Comment:
still wondering if the excessive use of v-if is required. You have default values and validators.